contrib/destination-starrocks/build.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
     implementation 'com.alibaba:fastjson:1.2.75'
 
     implementation project(':airbyte-config-oss:config-models-oss')
-//    implementation project(':airbyte-protocol:protocol-models')
+    // implementation project(':airbyte-protocol:protocol-models')
```

Comment:
Why annotate this protocol?

Reply:
It was already annotated before, I just moved the position of the slashes.

Reply:
You can remove it if not needed.

Reply:
It has been removed.